orbyter-ml-dev/Dockerfile

```diff
@@ -1,30 +1,13 @@
-# orbyter-ml-dev:3.5
-FROM manifoldai/orbyter-base-sys:3.2
+# orbyter-ml-dev:4.0
+FROM manifoldai/orbyter-base-sys:4.0
 COPY . /build
 WORKDIR /build
-RUN apt-get update && \
-    apt-get install -y libpq-dev && \
-    apt-get install -y graphviz && \
+RUN apt-get update -qq && \
+    apt-get install -qqy --no-install-recommends \
+        libpq-dev \
+        graphviz && \
     pip install -r requirements.txt && \
-    jupyter contrib nbextension install --user && \
-    jupyter nbextension enable --py widgetsnbextension && \
-    jupyter labextension install @jupyter-widgets/jupyterlab-manager && \
-    apt-get install -y --no-install-recommends \
-        xvfb \
-        libgtk2.0-0 \
-        libxtst6 \
-        libxss1 \
-        libgconf-2-4 \
-        libnss3 \
-        libasound2 && \
-     mkdir -p /home/orca && \
-     cd /home/orca && \
-     wget https://github.com/plotly/orca/releases/download/v1.2.1/orca-1.2.1-x86_64.AppImage && \
```

Comment:
@rkoppula as the plotly expert, is this still necessary?

Reply:
WE can remove this, but save it as a gist or something. We added that for some specific image rendering with plotly (not the interactive charts that we typically use)

Reply:
It'll forever be preserved in our git history :)